google-cloud-spanner/README.md

```diff
@@ -58,6 +73,33 @@ module GRPC
 end
 ```
 
+## FAQ for freezing development of `database` and `instance` classes
```

Comment:
What about replacing "classes" with "APIs"? I would also probably capitalize "database" and "instance" here to indicate that you are referring to types.

Reply:
I capitalized it. But didn't replace it with APIs. I was worried that users will interpret as we're deprecating the v1 APIs for Databases and Instances which is a big deal. Let me know if I don't understand the teminology though.

Reply:
LGTM!